change/@ni-nimble-blazor-b9a4025f-2fd1-4b3e-8295-540f02450064.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Seems to be comment posting races so FYI make sure discussions resolved in this thread: https://github.com/ni/nimble/pull/2080#discussion_r1593182562